specification/attributes/column_naming_convention.md

```diff
@@ -19,16 +18,20 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
-* Column IDs MUST NOT use abbreviations.
-* Column IDs SHOULD NOT use acronyms.
-* Column IDs MUST be alphanumeric with no special characters.
-* Columns that have an ID and a Name MUST have the Id or Name suffix in the Column ID. Display Name for a Column MAY
-  avoid the Name suffix if it is considered superfluous
+- All columns defined by FOCUS MUST follow the following rules:
+  - Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
+  - Column IDs MUST NOT use abbreviations.
+  - Column IDs SHOULD NOT use acronyms.
+  - Column IDs MUST be alphanumeric with no special characters.
+  - Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the `Name` suffix if there are no other columns with the same name prefix.
+- Custom columns SHOULD follow the same rules as FOCUS columns listed above.
+- All custom column IDs MUST start with a vendor prefix string and an underscore (e.g., `abc_ColumnName`).
```

Comment:
My preference would go to slash '/' or colon ': ' rather underscore '_', because underscore would make column names look like a mix of Snake case and Pascale case.
That is my first impression, but maybe I will get used to  '_'.
Curious to know what other think.

Reply:
The concern there is that most databases don't support them. Look at the supporting content. All the considerations are there. That said, I know slash will likely come in when we do grouping, so that's also something to consider.

Reply:
Ok, but some column names of some databases like Oracle are case insensitive by default, while FOCUS uses Pascal case.
Moreover, AWS columns in CUR contain slash.
So I believe data ingestion would do some transformation anyway to cope with database naming constraints.
But Ok I understand the rationale, thanks!